FIX:Change data type

hanzzb opened the pull request:
Change comparison method of input value to 'strcmp'.
Change data type to receive input as a string like below.

const char* gotoGame = "1";
const char* gotoLeaderBoard = "2";
const char* quitGame = "3";
char menu_option[10];
char cho[10];
char link[10];
char dec[10];

hanzzb committed:
FIX:Change data type

Change comparison method of input value to 'strcmp'.
Change data type to receive input as a string like below.

const char* gotoGame = "1";
const char* gotoLeaderBoard = "2";
const char* quitGame = "3";
char menu_option[10];
char cho[10];
char link[10];
char dec[10];

seohyeonKK commented:
잘 수정된 것 같아요 merge합시다

cherry2030 commented:
merge갑시다..!!

mjseok commented on Code.c:
```diff
@@ -316,35 +318,35 @@ void showBoard(char game_board[], Player_Info game_player1, Player_Info game_pla
 	printf("For more clarifications press Y else type any other character:- ");
-	scanf("%c", &link);
-	if (link == 'y' || link == 'Y')
+	scanf("%s", link);
```
이부분에 '%[^\n]s' 로 바꾸면 enter입력도 가능해서 바꾸는게 더 좋을 것 같아요~!

seohyeonKK replied:
동의합니다^^

hanzzb replied:
좋은 의견 감사합니다. 수정했습니다~~

hanzzb committed:
FIX:Allow 'enter' as input value

Correct like below because 'enter' is not recognized as an input value.

scanf("%[^\n]s", link);
